pkgs/by-name/en/ente-auth/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  appimageTools,
+  fetchurl,
+  lib,
+}:
+let
+  pname = "ente-auth";
+  version = "3.0.18-beta.2";
```

Comment:
Any reason this is packaging a beta instead of the latest release `3.1.1`?

Reply:
Yes, as when I made this PR ``3.0.18-beta.2`` was the latest available version.